### ISSUE TITLE
[BUG] Disabled labels fail WCAG 1.4.3 colour contrast

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#113]
Labels for Disabled fields are not except from WCAG 1.4.3.

1.4.3 - Contrast - The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:
Large Text - Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;
Incidental - Text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
Logotypes - Text that is part of a logo or brand name has no contrast requirement.

**Expected behavior**
The label text should not be changed and add "(Disabled)" to the label.

![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/3b982852-9bb3-4084-9638-70f36754bad0)

**Comment by ecstaticrainbow** [comment]
Duplicate of #160
